next-sitemap.config.js

```diff
@@ -49,6 +49,10 @@ module.exports = {
         disallow: [
           "/404",
           "/500",
+          "/*Browse.aspx$",
```

Comment:
I would also use the * instead of $ on these 2 lines just in case

Reply:
Done - I modified these routes to be wildcard routes